### PR TITLE
Set beacon node docker default to home folder

### LINKED ISSUE #342: Set default path of beacon chain data to user specific aplication directory with beacon subdir

### PR DESCRIPTION
**Pull request by BeroBurny**
closes #342

### CODE REVIEW

File: src/renderer/components/ConfigureBeaconNode/ConfigureBeaconNode.tsx [hunk 1]
```diff
@@ -28,7 +28,7 @@ export const ConfigureBeaconNode: React.FunctionComponent<IConfigureBNProps> = (
     const [networkIndex, setNetworkIndex] = useState(0);
     const defaults = getDefaultsForClient(props.clientName);
 
-    const defaultChainDataDir = path.join(getConfig(remote.app).storage.dataDir);
+    const defaultChainDataDir = path.join(getConfig(remote.app).storage.home) + "/beacon";
```
Comment (mpetrunic) [inline src/renderer/components/ConfigureBeaconNode/ConfigureBeaconNode.tsx:31]: I don't think we should store chain data in ~/beacon. It's wrong on so many levels 😄 
I think according to xdg, you should use `path.join(getConfig(remote.app).storage.dataDir, "beacon");`

https://www.electronjs.org/docs/api/app#appgetpathname

**Review by mpetrunic — CHANGES_REQUESTED** [review]

File: src/renderer/components/ConfigureBeaconNode/ConfigureBeaconNode.tsx [hunk 2]
```diff
@@ -28,7 +28,7 @@ export const ConfigureBeaconNode: React.FunctionComponent<IConfigureBNProps> = (
     const [networkIndex, setNetworkIndex] = useState(0);
     const defaults = getDefaultsForClient(props.clientName);
 
-    const defaultChainDataDir = path.join(getConfig(remote.app).storage.dataDir);
+    const defaultChainDataDir = path.join(getConfig(remote.app).storage.home, "beacon");
```
Comment (mpetrunic) [inline src/renderer/components/ConfigureBeaconNode/ConfigureBeaconNode.tsx:31]:
```suggestion
    const defaultChainDataDir = path.join(getConfig(remote.app).storage.dataDir, "beacon");
```

**Review by mpetrunic — CHANGES_REQUESTED** [review]

File: src/renderer/components/ConfigureBeaconNode/ConfigureBeaconNode.tsx (hunk 2 above)
Comment (BeroBurny) [reply src/renderer/components/ConfigureBeaconNode/ConfigureBeaconNode.tsx:31]: so are we gonna abandon this task?

Comment (mpetrunic) [reply src/renderer/components/ConfigureBeaconNode/ConfigureBeaconNode.tsx:31]: I think we still need beacon subdir

**Review by mpetrunic — APPROVED** [review]

**Pull request merged**